package.json

```diff
@@ -87,6 +87,7 @@
     "q": "1.0.1",
     "qrcode.react": "0.6.1",
     "qs": "4.0.0",
+    "ramda": "0.22.1",
```

Comment:
> I can appreciate your personal preference toward some of the design decisions of Ramda vs. Lodash, but I can't get on board with introducing this as a dependency due to the added bloat and onboarding confusion of two competing utility libraries.

We shouldn't ignore the added cost and complexity of an extra library for machines and developers, but we need to balance that with some way to try tech out, and switch if it's better.